src/iuse_actor.cpp

```diff
@@ -361,8 +361,24 @@ void consume_drug_iuse::load( JsonObject &obj )
 
 void consume_drug_iuse::info( const item&, std::vector<iteminfo>& dump ) const
 {
+    std::string vits;
+    for( const auto &v : vitamins ) {
+        // only display vitamins that we actually require
+        if( g->u.vitamin_rate( v.first ) > 0 ) {
+            if( !vits.empty() ) {
+                vits += ", ";
+            }
+            int lo = int( v.second.first  / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
+            int hi = int( v.second.second / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
```

Comment:
> and there's no guarantee that compilers optimize that

`vitamin_rate` is a `const` method so it's possible (probable?) and in any case that would be a good example of extreme premature optimization given that this function displays (once) the stats for 5 vitamins provided by an item